lib/pry/prompt.rb

```diff
@@ -19,7 +19,7 @@ class Pry
   #   # In [4]:
   # @since v0.11.0
   # @api public
-  module Prompt
+  class Prompt
```

Comment:
if you used a Struct here you wouldn't break API compatibility, since it also supports access through `#[]`.